### PR TITLE
Lovelace: add visible example for views

### PR DESCRIPTION
**Pull request by iantrich**
**Description:**

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/lovelace/views.markdown [hunk 1]
```diff
@@ -64,10 +64,125 @@ If you define `visible` as objects instead of a boolean to specify conditions fo
...
+
+You can specify the visibility of views as a whole or per-user.
+
+### Example
+
+```yaml
+views:
+  - title: Ian
+    visible:
+      - user: iantrich_id
```
Comment (bramkragten) [inline source/lovelace/views.markdown]: Use a hex string, so people know they have to search for a long weird string.

File: source/lovelace/views.markdown [hunk 2]
```diff
@@ -64,10 +64,125 @@ If you define `visible` as objects instead of a boolean to specify conditions fo
...
+### Example
+
+```yaml
+- title: Garden
+  icon: mdi:flower
+```
+
+## Visible
+
+You can specify the visibility of views as a whole or per-user.
```
Comment (bramkragten) [inline source/lovelace/views.markdown]: Maybe explain that it just isn't visible in the tab bar, but you can still reach it by url?

File: source/lovelace/views.markdown (hunk 1 above)
Comment (iantrich) [reply source/lovelace/views.markdown]: done

File: source/lovelace/views.markdown (hunk 2 above)
Comment (iantrich) [reply source/lovelace/views.markdown]: done

**Review by bramkragten — APPROVED** [review]

**Pull request merged**